### PR TITLE
Clean up translations list

### PR DESCRIPTION
**Pull request by safwansamsudeen**
Replaces and closes #897.

As discussed, I have moved the docs links to the Language column, have the names link to email addresses and GH usernames link to GH profiles. I've also added commas separating the items in the Links column and standardized the Transifix link.

I've added a few names of the language managers. For Korean, when I added the name 오동권, the table didn't format, for some reason. For those whom I haven't linked the email ID or GH profile, I couldn't find either.

There was a recent merge conflicting with this one: what's the difference between RTD and the normal docs? As #1114 had both in the Turkish row, I've maintained that. I've used Git for years and still mess up merging, so excuse me for the mess.

Is there anything else I could change?

----
:books: Documentation preview :books:: https://cpython-devguide--1119.org.readthedocs.build/documentation/translating/

### CODE REVIEW

**Comment by safwansamsudeen** [comment]
Thanks for the review!

**Comment by ezio-melotti** [comment]
It's likely a bug in sphinx-lint, since it doesn't play well with tables: https://github.com/sphinx-contrib/sphinx-lint#known-issues
In theory it should detect and skip tables, but it looks like it's not working in this instance.  I'm taking a look at the issue to see if I can put together a quick fix.

**Comment by hugovk** [comment]
Maybe switch to the list style format? Would make future maintenance easier too, as there's no need to shuffle the spacing next time.

**Comment by ezio-melotti** [comment]
This was discussed in https://github.com/python/devguide/pull/897#issuecomment-1588858310.
I now created a PR to fix the `sphinx-lint` issue: https://github.com/sphinx-contrib/sphinx-lint/pull/65

**Comment by hugovk** [comment]
Fair enough :)  List style would be easy to edit online via the GitHub UI (desktop and phones), Notepad, BBEdit, vim etc. (Although I bet vim has some plugin :) Anyway, thanks for the sphinx-lint PR and table style is fine.

**Comment by safwansamsudeen** [comment]
@CAM-Gerlach, just bumping this. I'm not sure after how long we're supposed to remind you (in other OSS projects I read that you can tag reviewers after a week), so do correct me in this regard.

**Comment by hugovk** [comment]
We need the CI to pass before we can merge.

That means either waiting for https://github.com/sphinx-contrib/sphinx-lint/pull/65 to be merged, released, and updated in this repo. Or switching to list style.

**Comment by ezio-melotti** [comment]
https://github.com/sphinx-contrib/sphinx-lint/pull/65 got merged, but it's not released yet.

**Comment by hugovk** [comment]
> We need the CI to pass before we can merge.
> 
> That means either waiting for [sphinx-contrib/sphinx-lint#65](https://github.com/sphinx-contrib/sphinx-lint/pull/65) to be merged, released, and updated in this repo. Or switching to list style.

@safwansamsudeen This is now done, please can you resolve the conflicts in this PR? Thanks!

**Comment by safwansamsudeen** [comment]
@hugovk I'm busy this week, but I think I'll be able to do it by the end of the month. I hope that's alright.

**Comment by hugovk** [comment]
Perfectly alright, no rush at all! Thank you!

**Comment by hugovk** [comment]
@safwansamsudeen Hi, checking in to see if you've time to look at this again? Thanks!

**Comment by safwansamsudeen** [comment]
@hugovk I'm sorry, but I don't think I'll be able to continue working on this.

**Comment by hugovk** [comment]
That's okay, thanks for your work so far!

**Comment by hugovk** [comment]
One question: should we add a link in the "Language" column for all the translations which are on docs.python.org, even if they're not in the dropdown menu?

For example, right now https://devguide.python.org/documentation/translating/ has https://docs.python.org/pl/ but it's not in the dropdown.

But it also doesn't have https://docs.python.org/uk/ (also not in the dropdown).

File: documentation/translating.rst [hunk 1]
```diff
@@ -12,140 +12,94 @@ They are built by `docsbuild-scripts
...
+       `article <https://rgth.co/blog/python-ptbr-cenario-atual/>`__
+   * - Russian (ru)
+     - Daniil Kolesnikov (:github-user:`MLGRussianXP`, `email <mailto:mlgrussianxp@gmail.com>`__)
+     - :github:`GitHub <MLGRussianXP/python-docs-ru>`,
+       `mail <https://mail.python.org/pipermail/doc-sig/2019-May/004131.html>`__
+   * - `Simplified Chinese (zh-cn) <https://docs.python.org/zh-cn/>`__
+     - Shengjing Zhu (:github-user:`zhsj`),
+       Du, Meng (:github-user:`dumeng`)
+     -  :github:`GitHub <python/python-docs-zh-cn>`,
+        `Transifex <tx_>`_
```
Comment (ezio-melotti) [inline documentation/translating.rst:85]:
```suggestion
     - :github:`GitHub <python/python-docs-zh-cn>`,
       `Transifex <tx_>`_
```

**Comment by ezio-melotti** [comment]
> One question: should we add a link in the "Language" column for all the translations which are on docs.python.org, even if they're not in the dropdown menu?

I don't see why not.  Translators might want to check that out, and the link will make it easier for them to access it, especially if it's not listed in the dropdown.

**Review by ezio-melotti — APPROVED** [review]

**Pull request merged**